assets/files/etc/sysconfig/network-scripts/ifcfg-ens4

```diff
@@ -0,0 +1,5 @@
+DEVICE=ens4
+BRIDGE=brext
```

Comment:
We've resolved the hard-coded device name in 98-brextscript but I'm wondering how we can move away from this hard-coded file?

Reply:
my bad, this is a left over, this file is no longer added to the nodes and we instead generate it on the file through the brext script